components/reducers/entitiesReducer.js

```diff
@@ -0,0 +1,99 @@
+export const initialState = {
+  values: [],
+  columns: [],
+  rows: [],
+  entity_type_name: null,
+  isLoading: false,
+  isRefresh: true,
+  entry: 10,
+  page: 0,
+  rows:null,
+  setsRenderer:0,
+  firstFetch:true,
+};
+
+
+export const LOADING = "LOADING";
+export const REFRESH = "REFRESH";
+export const CLEANUP = "CLEANUP";
+
+export const SET_VALUES = "SET_VALUES";
```

Comment:
You can refactor this to a separate file. I suggest you add this to reducers > actions.js like this

<img width="350" alt="Screenshot 2023-02-06 at 1 12 07 PM" src="https://user-images.githubusercontent.com/101687552/216888862-48d408d7-d8ca-490c-87cc-0f4253a2812a.png">